packages/shared/src/react/hooks/useReverification.ts

```diff
@@ -180,10 +196,7 @@ type UseReverificationResult<
  * }
  * ```
  */
-function useReverification<
-  Fetcher extends (...args: any[]) => Promise<any> | undefined,
-  Options extends UseReverificationOptions,
->(fetcher: Fetcher, options?: Options): UseReverificationResult<Fetcher, Options> {
+export const useReverification: UseReverification = (fetcher, options) => {
```

Comment:
I moved the type definitions from inline into `UseReverification` to make it easier to read